src/CrudTrait.php

```diff
@@ -17,7 +17,8 @@
     public static function getPossibleEnumValues($field_name)
     {
         $instance = new static(); // create an instance of the model to be able to get the table name
-        $type = DB::select(DB::raw('SHOW COLUMNS FROM `'.Config::get('database.connections.'.env('DB_CONNECTION').'.prefix').$instance->getTable().'` WHERE Field = "'.$field_name.'"'))[0]->Type;
+        $connectionName = $instance->getConnectionName();
+        $type = DB::connection($connectionName)->select(DB::raw('SHOW COLUMNS FROM `'.Config::get('database.connections.'.$connectionName.'.prefix').$instance->getTable().'` WHERE Field = "'.$field_name.'"'))[0]->Type;
```

Comment:
It's possible we should do some escaping in this raw query (but maybe I'm being paranoid?